### PR TITLE
Ensure same default hostname and service path is used

### PR DESCRIPTION
**Pull request by JohnLeyva**
Ensure same default hostname and service path is used in dotnet framework and core clients

Accepts --hostname cli argument

### CODE REVIEW

File: src/Samples/NetCoreClient/Program.cs [hunk 1]
```diff
@@ -6,16 +6,17 @@ namespace NetCoreClient
 {
     class Program
     {
-        private static readonly string s_hostname = "localhost";
-
-        static void Main()
+        /// <param name="hostname">use commanline argument --hostname localhost
+        /// or something similar to indicate the WCF Server hostname </param>
+        static void Main(string hostname)
```
Comment (mconnew) [inline src/Samples/NetCoreClient/Program.cs:11]: I'm not comfortable pulling in an alpha release package for command line parsing for a single parameter. I would rather just check if args.Length == 1, then set hostname to that. So something like:
```c#
string hostname = args.Length == 1 ? args[0] : null;
```
It's a nice concept, but I had a look at their code and there's a few problems, like consuming an extra thread unnecessarily for the duration of the app. I know this is samples, but I don't want to encourage other people to use an alpha package.

File: src/Samples/NetCoreClient/Program.cs [hunk 2]
```diff
@@ -6,16 +6,17 @@ namespace NetCoreClient
...
         {
-            Settings settings = new Settings().SetDefaults(s_hostname, "EchoService");
+            Console.Title = "WCF .Net Core Client";
```
Comment (mconnew) [inline src/Samples/NetCoreClient/Program.cs:13]: If this is run on the command line, will this permanently change the console title or will it revert once the process exits?

File: src/Samples/NetCoreClient/Program.cs (hunk 1 above)
Comment (JohnLeyva) [reply src/Samples/NetCoreClient/Program.cs:11]: Makes sense, I'll remove it

File: src/Samples/NetCoreClient/Program.cs (hunk 2 above)
Comment (JohnLeyva) [reply src/Samples/NetCoreClient/Program.cs:13]: When the process exits, the console title reverts to the previous value

**Comment by JohnLeyva** [comment]
Please let me know if something is still pending

**Comment by mconnew** [comment]
This looks good. Sorry it dropped off my radar for a while.

**Pull request merged**